web/src/beta/features/ProjectSettings/innerPages/PublicSettings/PublicSettingsDetail.tsx

```diff
@@ -244,6 +244,7 @@ const PublicSettingsDetail: React.FC<Props> = ({
               theme={currentTheme}
               accessToken={accessToken}
               onNotificationChange={onNotificationChange}
+              version={"visualizer"}
```

Comment:
I think it's reasonable to passing this, but we need to keep in mind this extension is not from marketplace, it's from reearth cloud. we need to refactor this extension in the future.

Reply:
Got it. Thanks